tests/mock_sftpserver.cpp

```diff
@@ -30,6 +30,7 @@ extern "C"
     IMPL_MOCK_DEFAULT(2, sftp_reply_handle);
     IMPL_MOCK_DEFAULT(1, sftp_get_client_message);
     IMPL_MOCK_DEFAULT(1, sftp_client_message_free);
+    IMPL_MOCK_DEFAULT(1, sftp_client_message_get_submessage);
```

Comment:
Looks like this is not needed based on the tests. 

If it's needed in the future, it also needs to be added to "c_mock_defines.cmake"